sass/components/_emoticons.scss

```diff
@@ -79,7 +79,7 @@
     margin-right: 3px;
     position: absolute;
     width: 315px;
-    z-index: 999;
+    z-index: 1100;
```

Comment:
Is this having to change to get the picker to appear on top of the status dropdown?

Reply:
Yes. We had to make:

`z-index(EmojiPicker) > z-index(CustomStatusModal / GenericModal)`

to be able to select emojis from the custom status modal.

Reply:
Fun. Modals and popovers are another thing we need to tackle when it comes to rewriting the menu components like I mentioned before, so hopefully we'll get to a point where we don't need to fight with z-indexes any more.